### ISSUE TITLE
Custom link title bug?

### ISSUE THREAD

**Issue by fremontmj** [org-roam/org-roam#1358]
I have set org-roam-link-title-format to "§%s". This worked fine until I updated Doom Emacs to the latest version, which includes org-roam release be95b42.

Now whenever I change the title of a note, all the links to that note add an additional § in front of the file name, so I get links that look like "§§§§§§§test note5" (after changing the note name several times).

I can delete the extra characters and the link still works, but this is an untenable solution...

I don't believe this happened before the update.

**Comment by jethrokuan** [comment]
I don't think there's a good way to support `org-roam-link-title-format` with the automatic file renaming from title.

You'll probably want to set `org-roam-rename-file-on-title-change` to nil.

**Comment by fremontmj** [comment]
Thanks. I made the mistaken assumption that org-roam-link-title-format was simply a formatting made at display-time. I didn't realize there was some (I guess) stored component to it.

Until I applied the update, I had been using an earlier version of org-roam that didn’t automatically change link names when their target name changed. That's a nice enhancement!

By the way, org-roam-rename-file-on-title-change is not mentioned in the manual.

Best.
